git-extra/git-extra.install

```diff
@@ -219,9 +219,14 @@ test -d "$TMPDIR" || test ! -d "$TMP" || {\
 	# Re-enable at least AES256-cbc and AES128-cbc, as they are still used
 	# e.g. in VSTS
 	test ! -f /etc/ssh/ssh_config ||
-	grep -q '^Ciphers .*aes256-cbc,aes192-cbc' /etc/ssh/ssh_config ||
-	sed -i -e '/^[# ]*Ciphers /{s/^# *//;s/$/,aes256-cbc,aes192-cbc/}' \
-		/etc/ssh/ssh_config
+	{
```

Comment:
This curly bracket (and the corresponding indentation) is not really needed, is it?

Reply:
We have a similar situation compared to above in https://github.com/git-for-windows/build-extra/pull/192/files#diff-5a39db150c328ad0b5127f77720de3edR161. If /etc/ssh/ssh_config exists, then we want to do two things: revert change by prioer versions of git-extra and append the new incremental ciphers list.

But we can do this instead, if you prefer:

```
test ! -f /etc/ssh/ssh_config ||
! grep -q '^Ciphers [a-z].*cbc' ||
sed -i -e 's/^Ciphers [a-z].*cbc/# &/' /etc/ssh/ssh_config

test ! -f /etc/ssh/ssh_config ||
grep -q '^Ciphers\s' /etc/ssh/ssh_config ||
sed -i -e '$a# Added by git-extra\nCiphers +aes128-cbc,3des-cbc,aes256-cbc,aes192-cbc' \
    /etc/ssh/ssh_config
```

Reply:
Oooops! I totally missed that there was still a `test ! -f /etc/ssh/ssh_config ||` in front of the curlies! Please accept my apologies, my review was incorrect.

So maybe

```sh
test ! -f /etc/ssh/ssh_config || {
    ! grep -q '^Ciphers [a-z].*cbc' ||
    sed -i -e 's/^Ciphers [a-z].*cbc/# &/' /etc/ssh/ssh_config

    grep -q '^Ciphers\s+' /etc/ssh/ssh_config ||
    printf '%s\n%s\n' \
        '# Added by git-extra'
        'Ciphers +aes128-cbc,3des-cbc,aes256-cbc,aes192-cbc' \
        >/etc/ssh/ssh_config
}
```

Reply:
Not at all. Thank you for the review. Indeed, the extra blank line makes this more readable, and I aligned the formatting of the curly braces as suggested.